### PR TITLE
feat(pci.data-processing): dashboard - add TTL

### PR DESCRIPTION
**Pull request by jperchoc**
DATATR-10

Signed-off-by: Jonathan Perchoc <jonathan.perchoc@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/datatr-3-dataprocessing-jobs-dashboard f`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DATATR-10
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Add TTL to the dashboard

## Related

```
DATATR-3
  ^ 
  | - DATATR-6
  | - DATATR-7
  | - DATATR-10
  | - DATATR-11
  | - DATATR-12
  | - DATATR-13
```
[DATATR-3](https://github.com/ovh/manager/tree/feat/datatr-3-dataprocessing-jobs-dashboard)
[DATATR-6](https://github.com/ovh/manager/pull/7120)
[DATATR-7](https://github.com/ovh/manager/pull/7115)
[DATATR-10](https://github.com/ovh/manager/pull/7116)
[DATATR-11](https://github.com/ovh/manager/pull/7119)
[DATATR-12](https://github.com/ovh/manager/pull/7117)
[DATATR-13](https://github.com/ovh/manager/pull/7118)

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   "data_processing_details_configuration_python_file_label": "Fichier Python",
   "data_processing_details_configuration_main_class_label": "Classe principale",
   "data_processing_details_configuration_arguments_label": "Arguments",
+  "data_processing_details_configuration_ttl_label": "Time To Live (TTL)",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/translations/Messages_fr_FR.json:26]: **question:** Are we expecting any translation based on the initial content?

If not, then we are relying on static content that can be stored as a constant or directly added into the template (if possible).

File: packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/job-dashboard.html [hunk 2]
```diff
@@ -207,6 +207,13 @@
                 data-description="{{ $ctrl.job.engineParameters.arguments }}"
             >
             </oui-tile-definition>
+            <oui-tile-definition
+                ng-if="$ctrl.job.ttl"
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/job-dashboard.html:211]:
```suggestion
                data-ng-if="$ctrl.job.ttl"
```

Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/job-dashboard.html]: Done here: https://github.com/ovh/manager/pull/7116/commits/82042c2a0a5a9e5d0eefe3656d93bd7ead21eea9

File: packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/translations/Messages_fr_FR.json (hunk 1 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/translations/Messages_fr_FR.json]: Removed from translation files and added directly to the template here: https://github.com/ovh/manager/pull/7116/commits/82042c2a0a5a9e5d0eefe3656d93bd7ead21eea9

**Review by antleblanc — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**